### ISSUE TITLE
about difference between paper and this repo

### ISSUE THREAD

**Issue by ltm920716** [lllyasviel/ControlNet#455]
Hi，
    The model structure and some params in this repo are different from the paper，so the pre-trained stable-diffusion model used for the paper is a private model trained by yourself？Could you please tell some tricks like what type of vae is better for controlnet？thanks！

**Comment by ltm920716** [comment]
in my experiment，the loss does not down to stable，but the model dose show effect，so the loss is does not matter？
